config/initializers/version.rb

```diff
@@ -0,0 +1 @@
+TestGuru::VERSION = '0.0.2'.freeze
```

Comment:
я бы скорее сказал, что это 
```suggestion
TestGuru::VERSION = '0.1.1'.freeze
```
второй разряд - первый урок
третий разряд - первый патч после начальной версии (первая пересдача)